notebooks/forest_inference_demo.ipynb

```diff
@@ -516,11 +471,12 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = df.persist(); wait(df);"
+    "df = df.persist(); \n",
```

Comment:
Oh, and don't forget to drop the semicolons, too.

Reply:
thanks should be all set!